eventlet/backdoor.py

```diff
@@ -10,6 +10,7 @@
 import eventlet
 from eventlet import hubs
 from eventlet.support import greenlets, get_errno
+from eventlet.support.afinet import ip_defaults
```

Comment:
Only module imports are fine and safe. `from module import name` creates multiple references to same object which are impossible to patch and harder to understand while reading code. I know this may sound hypocritical with previous line importing `get_errno`, please ignore that's legacy code.